index-navbar.html

```diff
@@ -0,0 +1,183 @@
+<!DOCTYPE html>
+<html lang="nl">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <!--fonts-->
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
+    <!--CSS stylesheets-->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <link rel="stylesheet" href="stylesheets/../style.css">
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/summernote/0.8.12/summernote-bs4.css" rel="stylesheet">
+
+    <!--JS-->
+    <script type="text/javascript" src="scripts.js"></script>
+    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.11.0/umd/popper.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta/js/bootstrap.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/summernote/0.8.12/summernote-bs4.js"></script>
+
+    <title>Showkorps Concento</title>
+</head>
+
+<body>
+
+    <div class="section-container">
+
+        <!--menu nav-->
+        <div class="section-navbar">
+            <nav class="topnav">
+                <div class="row">

```

Comment:
When using Bootstrap's grid system (`.row` and `.col-*`) `.container` or `.container-flud` wrappers are required. See: https://getbootstrap.com/docs/4.3/layout/overview/

There is a lot of mixed content. I would suggest to decide to always use Bootstrap's grid system or don't use it at all. Use it - to learn Bootstrap and make website responsive bydefault. Don't use it to focus on learning CSS grid/flexbox.

Reply:
https://github.com/innersilence67/showkorps-concento-website/issues/3

change is too big for this branch. I will make this one in new branch as there will be a lot of changes not related to this branch